src/netstandard/ref/System.cs

```diff
@@ -3373,18 +3373,9 @@ public partial class Random
         public override bool Equals(object value) { throw null; }
         public bool Equals(System.Range other) { throw null; }
         public override int GetHashCode() { throw null; }
-        public System.Range.OffsetAndLength GetOffsetAndLength(int length) { throw null; }
+        public (int Offset, int Length) GetOffsetAndLength(int length) { throw null; }
```

Comment:
Is PascalCasing for tuple element names an "official" style policy anywhere?

I have no problem with that being the naming convention, but as the names are part of the public API, I'd really like to make sure we don't end up with a mixture of styles. As the previous policy was "do not put tuples in the public API" this wouldn't have come up before.

Reply:
> Is PascalCasing for tuple element names an "official" style policy anywhere?

It's the style we've agreed to (equivalence with public fields and properties).  We're working on writing up and publishing guidance (for tuples, and the other concept areas newer than .NET Framework 3.5)

Reply:
Great - thanks very much for confirming :)

Reply:
@jskeet 

Yes. We've met twith the C# team and that's were we landed. As @bartonjs said, we're working on publishing naming guidlines.